README.md

```diff
@@ -445,46 +445,57 @@ Note that if you want to pass the token name as a hex or an int, you need to pro
 
 ## Extending Nile with plugins
 
-Nile has the possibility of extending its CLI and `NileRuntimeEnvironment` functionalities through plugins. For developing plugins for Nile fork [this plugin example](https://github.com/franalgaba/nile-plugin-example) boilerplate and implement your desired functionality with the provided instructions.
+Nile has the possibility of extending its CLI and `NileRuntimeEnvironment` functionalities through plugins. For developing plugins you can fork [this example template](https://github.com/franalgaba/nile-plugin-example) and implement your desired functionality with the provided instructions.
 
 ### How it works
 
-This implementation takes advantage of the native extensibility features of [click](https://click.palletsprojects.com/). Using click and leveraging the Python [entrypoints](https://packaging.python.org/en/latest/specifications/entry-points/) we have a simple manner of handling extension natively on Python environments through dependencies. The plugin implementation on Nile looks for specific Python entrypoints constraints for adding commands.
+This implementation takes advantage of the native extensibility features of [click](https://click.palletsprojects.com/). Using click and leveraging the Python [entry points](https://packaging.python.org/en/latest/specifications/entry-points/) we have a simple manner of handling extension natively on Python environments through dependencies. The plugin implementation on Nile looks for specific Python entry points constraints for adding commands for either CLI or NRE.
 
-In order for this implementation to be functional, it is needed by the plugin developer to follow some development guidelines defined in this simple plugin example extending Nile for a dummy greet extension. In a brief explanation the guidelines are as follows:
+In order for this implementation to be functional, it is needed by the plugin developer to follow some guidelines:
 
-1. Define a Python module that implements a click command or group:
+1. Use click if the plugin provides a CLI command:
```

Comment:
What do you think about treating click as a proper noun and capitalizing it? => "Use Click if..."
Or wrapping it in backticks? => "Use `click` if..."

Reply:
Looks better for sure.

Reply:
Actually, I went to do the change, and I'm not so sure anymore, because we use the package name in these cases in the rest of the code (nile, nile-coverage, click, etc...). Using capitalization is a bit weird in cases like Nile-Coverage? Nile Coverage? And formatting the package name as code, doesn't look that good IMO. We should define a format for this, I think in the documentation task for the next milestone.

Reply:
> We should define a format for this, I think in the documentation task for the next milestone.

Yeah, good call. I think that's a better idea 👍

Reply:
Click [calls itself Click](https://click.palletsprojects.com/en/8.1.x/) so can do the same. I agree on a special treatment of some kind to highlight it's a proper noun.